vendor/bat-native-ads/include/bat/ads/ads.h

```diff
@@ -44,75 +51,77 @@ class ADS_EXPORT Ads {
   // Should be called to get the region for the specified locale
   static std::string GetRegion(const std::string& locale);
 
-  // Should be called when Ads are enabled or disabled on the Client
-  virtual void Initialize() = 0;
+  // Should be called when Ads is enabled on the Client
+  virtual void Initialize(InitializeCallback callback) = 0;
 
-  // Should be called when the browser enters the foreground
-  virtual void OnForeground() = 0;
+  // Should be called when Ads is disabled on the Client
+  virtual void Shutdown(ShutdownCallback callback) = 0;

```

Comment:
We will want to add this call here:

https://github.com/brave/brave-core/blob/40e6df39e1903e1240f54d97fe2d9a3168420c70/vendor/brave-ios/Ads/BATBraveAds.mm#L110-L117